express-get-json/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const array = [];
+
+for (const property in grades) {
+  array.push(grades[property]);
+}
+
+JSON.stringify(array);
```

Comment:
You want to do this within the callback function for your route. The data in the object could change so we want to build the array "fresh" every time a request is made to the server.